memiavl/db.go

```diff
@@ -171,8 +190,9 @@ func (db *DB) reloadMultiTree(mtree *MultiTree) error {
 }
 
 type snapshotResult struct {
-	mtree *MultiTree
-	err   error
+	mtree   *MultiTree
+	err     error
+	version uint32
```

Comment:
rename `version` to sth like `snapshotVersion`, to distinguish with latest version of `mtree` which has replay WAL.